tools/static-data/generate-azure-pipelines-marketplace-tasks.mjs

```diff
@@ -0,0 +1,12 @@
+import got from 'got';
```

Comment:
this file needs to be called from our auto update GitHub actions workflow

Reply:
Add an extra entry here:

https://github.com/renovatebot/renovate/blob/d638ed7f8898879bbb61efc029ea5eedc94dd742/package.json#L52-L55

Reply:
i think it should be combined with existing azure update script